### PR TITLE
Bundle gen

### PR DESCRIPTION
**Pull request by psykokwak4**
python sdk template is moved to sdk/python/profile

### CODE REVIEW

File: ydkgen/__init__.py [hunk 1]
```diff
@@ -30,97 +32,339 @@
...
+            output_dir (str): The output directory for generated APIs.
+            ydk_root   (str): The ydk root directory. Relative file names in
+                              the profile file are resolved relative to this.
+            groupings_as_class (bool): If set to true, YANG grouping is
+                                       converted to a class.
+            language (str): Language for generated APIs
+            pkg_type (str): Package type for generated APIs.
+                            Valid option for profile approach is: 'profile'.
+                            Valid options for bundle approach are: 'core',
+                                                                   'packages'.
```
Comment (abhikeshav) [inline ydkgen/__init__.py]: should this be 'bundle' instead of 'packages'?

**Comment by abhikeshav** [comment]
Since this is a major change to the way ydk-gen is going to be used, the documentation in [README.md](https://github.com/CiscoDevNet/ydk-gen/blob/master/README.md) needs to be changed to include the latest way to use generate.py.

File: ydkgen/__init__.py (hunk 1 above)
Comment (psykokwak4) [inline ydkgen/__init__.py]: Yes, it should be, sorry about that.

**Comment by psykokwak4** [comment]
Ok, will change [README.md](https://github.com/CiscoDevNet/ydk-gen/blob/master/README.md) in next commit.

**Pull request closed without merging**